src/workspace/Lesson/SectionOverview/Preview/Controls/index.jsx

```diff
@@ -104,44 +106,26 @@ const Controls = ({
     }
 
     async function confirmDelete() {
-        dispatch({
-            type: "PREPARE_DELETE_SECTION",
-            payload: sectionId,
-        })
-
-        setTimeout(() => {
-            dispatch({
-                type: "DELETE_SECTION",
-                payload: sectionId,
-            })
-        }, 250)
-
-        await api.delete(`/editor/sections/${sectionId}`)
+        dispatch(removeSection(sectionId))
     }
 
-    const isOnly = store.lesson.sections.length === 1
+    const isOnly = store.sections.length === 1
```

Comment:
Für was wird das benötigt?

Reply:
hab ich nicht geschrieben aber sieht nach irgendwas mit css aus: 
 ${props => {
        return (
            props.isOnly &&
            css`
                cursor: not-allowed;
                user-select: none;
                opacity: 0.5;
                &:active {
                    cursor: not-allowed;
                }
            `
        )

Reply:
Irgendwie gefällt mir das nicht. Können wir das umbennen?